filters/tee/tee.go

```diff
@@ -203,12 +208,33 @@ func (r *tee) Response(filters.FilterContext) {}
 // Request is copied and then modified to adopt changes in new backend
 func (r *tee) Request(fc filters.FilterContext) {
 	req := fc.Request()
+
+	// omit loops
+	if req.Header.Get(ShadowTrafficHeader) != "" {
+		s := "Skipper Shadow Traffic Loop detected"
+		fc.Serve(&http.Response{
+			StatusCode: 465,
+			Status:     s,
+			Header: http.Header{
+				"Content-Type":   []string{http.DetectContentType([]byte(s))},
```

Comment:
Should we just add content-type? It won't change

Reply:
Does it matter?
I just want an rfc compliant content type set and don't come back if some rfc changes what we should set 😊